aws/table_aws_ssm_managed_instance_compliance.go

```diff
@@ -0,0 +1,158 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ssm"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsSSMManagedInstanceCompliance(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ssm_managed_instance_compliance",
+		Description: "AWS SSM Managed Instance Compliance",
+		List: &plugin.ListConfig{
+			ParentHydrate: listSsmManagedInstances,
```

Comment:
@bigdatasourav I'm a bit worried that for large environments, having this table using parent hydrate will result in a lot of API calls, i.e., hybrid environments that have all of their on-prem instances managed by SSM, along with some EC2 instances too.

Can you please switch this table to require a key, `resource_id`, to reduce the number of calls?

We should then update any examples, and include an example where we join the managed instance results with this table as well to help users.